apps/devhub/tasks.py

```diff
@@ -181,11 +182,12 @@ def handle_upload_validation_result(results, upload_pk, annotate=True):
 def handle_file_validation_result(results, file_id, annotate=True):
     """Annotates a set of validation results, unless `annotate is false, and
     saves them to the given File instance."""
-    if annotate:
-        results = annotate_validation_results(results)
+    with statsd.timer('devhub.validator.handle_file_validation_result'):
```

Comment:
this one exists at https://app.datadoghq.com/metric/explorer?live=true&page=0&is_auto=false&from_ts=1452542602739&to_ts=1452629002739&tile_size=m&exp_metric=addons_prod.tasks.devhub.tasks.handle_file_validation_result.avg&exp_scope=&exp_agg=avg&exp_row_type=metric